css/style.css

```diff
@@ -115,12 +115,13 @@ footer {
   padding: 0 2rem;
 }
 
-.img-rounded {
+img {
```

Comment:
Not sure how I feel applying this to img tag in general, but probably fine for our purposes.

Reply:
Hmmmm u right, lemme think about how to do this better

Reply:
I changed it to `.content img` in both places... is this any better? 🙈

Reply:
Lol I think this is marginally better, but doesn't really tackle the issue. What was wrong with the class we used previously? (Also I honestly don't really care, since this will probably never present an issue)

Reply:
I fixed it! The issue was that I didn't know how to add the class to the image in markdown, but you can do it like

`![2E family photo](./img/2emain.jpg){:.img-rounded}`